manifests/profiles/version.yaml

```diff
@@ -130,3 +130,30 @@ controllerManager:
     requests:
       cpu: 100m
       memory: 25Mi
+
+mosquitto:
+  enable: true
+  image:
+    repository: eclipse-mosquitto
+    tag: "1.6.15"
+    pullPolicy: "IfNotPresent"
+    pullSecrets: []
+  labels:
+    k8s-app: eclipse-mosquitto
+    kubeedge: eclipse-mosquitto
+  annotations: {}
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: node-role.kubernetes.io/edge
+                operator: Exists
+  tolerations: []
+  resources:
```

Comment:
Have you tested this request and limits? Is there a possibility of OOM?

Reply:
The data of 64 bytes 'x' and time-formatted is send to mqtt broker every second
```bash
/ # mosquitto_sub -t "default/test2"
2023-12-05 15:55:27 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:28 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:29 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:30 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:31 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:32 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:33 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:34 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:35 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:36 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:37 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2023-12-05 15:55:38 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
...
```
Observe memory usage
```bash
root@ubuntu:~# nerdctl --namespace k8s.io ps
CONTAINER ID    IMAGE                                                        COMMAND                    CREATED           STATUS    PORTS    NAMES
211e0a5d0fde    docker.m.daocloud.io/kubeedge/eclipse-mosquitto:1.6.15       "/docker-entrypoint.…"     2 minutes ago     Up                 k8s://kubeedge/edge-eclipse-mosquitto-g2dkn/edge-eclipse-mosquitto
3e2539173fee    docker.m.daocloud.io/kubeedge/pause:3.6                      "/pause"                   32 minutes ago    Up                 k8s://default/2048-test-686bcf67cc-kj9tn

root@ubuntu:~# nerdctl --namespace k8s.io stats 211e0a5d0fde

CONTAINER ID   NAME      CPU %     MEM USAGE / LIMIT   MEM %     NET I/O          BLOCK I/O        PIDS
211e0a5d0fde   --        3.26%     1.699MiB / 64MiB    2.66%     2.95GB / 485MB   262kB / 25.6kB   3
```

We can see that resources are sufficient. If necessary, we can increase the limit.

Reply:
It appears that the resources allotted are adequate. Thank you for your thorough response.